roles/download/defaults/main.yml

```diff
@@ -143,6 +144,8 @@ contiv_ovs_image_repo: "contiv/ovs"
 contiv_ovs_image_tag: "latest"
 cilium_image_repo: "docker.io/cilium/cilium"
 cilium_image_tag: "{{ cilium_version }}"
+kube_router_image_repo: "cloudnativelabs/kube-router"
```

Comment:
Please add to the `downloads:` section in this file too

Reply:
Done.

Reply:
Done.